utilities/transactions/pessimistic_transaction_db.cc

```diff
@@ -545,6 +545,42 @@ Status PessimisticTransactionDB::Put(const WriteOptions& options,
   return s;
 }
 
+Status PessimisticTransactionDB::PutEntity(const WriteOptions& options,
+                                           ColumnFamilyHandle* column_family,
+                                           const Slice& key,
+                                           const WideColumns& columns) {
+  {
```

Comment:
What's the benefit when we have these in a local scope? One thing I can see is that we can `const` the `Status` and not re-use it, but I don't see the same pattern in other methods like `Put` and `Delete`, so asked.